unified-codes/apps/data-service/src/app/v1/types/DgraphDataSource.ts

```diff
@@ -156,9 +156,11 @@ export class DgraphDataSource extends RESTDataSource {
             description: name@*
           }
           children {
+            code
```

Comment:
Some codes were missing, guessing maybe it wasn't possible before but with all these new level mappings it's now possible? 🤷  was causing some items in mSupply to not show up

Reply:
mSupply is now directly grabbing entities at the `Unit` and `DoseStrength` levels, so I think all this stuff is no longer needed!

Reply:
Yup this stuff is all gone after merging in `master` 🥳